### PR TITLE
Homepage UI

### PR DESCRIPTION
**Pull request by pankaj-prog**
homepage screen :
Screenshot 2022-05-18 033038

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+/* util classes */
+
+.link {
+  text-decoration: none;
+  color: inherit;
+}
+
+.gutter-bottom-8 {
+  margin-bottom: 0.5rem;
```
Comment (tarunsankhla) [inline src/index.css:27]: avoid using less then 1 rem

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+.icon-btn-wrapper .btn {
+  padding: 4px;
+}
+
+.icon-btn-wrapper .name {
+  display: none;
+  opacity: 0;
+  background-color: var(--COLOR-SECONDARY-V3);
```
Comment (tarunsankhla) [inline src/index.css:75]: why do you need opacity as 0 when display is none

**Review by tarunsankhla — APPROVED** [review]
looks good to merge

File: src/App.js [hunk 3]
```diff
@@ -1,7 +1,46 @@
...
+  useEffect(() => {
+    if (location.pathname == "/") {
+      navigate("/home", { replace: true });
+    }
+  }, []);
+
+  return (
+    <Routes>
+      <Route path="/" element={<UserRoutes />}>
+        <Route path="home" element={<Home />} />
```
Comment (muskanbagrecha) [inline src/App.js:31]: I think home or explore can be a public route.

File: src/components/Post/Post.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+const Post = () => {
+  return (
+    <article className="post gutter-bottom-16">
+      <header className="post-header gutter-bottom-16">
+        <div className="post-user-details">
+          <span className="avatar avatar-round">
+            <img
+              className="responsive-img"
+              src="https://adaptiveui.netlify.app/src/assets/avatar-default.png"
+              alt="...."
```
Comment (muskanbagrecha) [inline src/components/Post/Post.jsx:14]: Please give a meaningful alt.

File: src/App.js (hunk 3 above)
Comment (muskanbagrecha) [inline src/App.js:31]: I think home or explore can be a public route.

**Review by muskanbagrecha — APPROVED** [review]
The code looks good. Just minor changes.

File: src/index.css (hunk 2 above)
Comment (pankaj-prog) [reply src/index.css:75]: to show the transition on hover

File: src/components/Post/Post.jsx (hunk 4 above)
Comment (pankaj-prog) [reply src/components/Post/Post.jsx:14]: for now, its just dummy data.. will add it while adding final db.

**Pull request merged**